### DISCUSSION TITLE
MFXDatePicker range date

### DISCUSSION THREAD

**Discussion by delgadov** [palexdev/MaterialFX#157] [Ideas]
Hello,

Is possible to set a range in an unique MFXDatePicker? Example,

First click: init date
Second click: final date

So set background color between init/final date in a unique MFXDatePiicker, is it possible?

Google image example
![image](https://user-images.githubusercontent.com/58339268/157683441-58c77587-d45f-4e8c-ad23-b86f49326152.png)

**Comment by palexdev** [comment]
That is a RangeDatePicker, and no, it's currently not possible with MaterialFX as the control is not yet implemented
